manifests/init.pp

```diff
@@ -136,7 +136,20 @@
   $proxy_type                                     = undef,
   $proxy_server                                   = undef,
   Hash $extensions                                = {},
-  Hash $settings                                  = {},
+  Hash $settings                                  = {
```

Comment:
I think we need at least one acceptance test to verify that those settings actually work.